web/src/navconfig.jsx

```diff
@@ -74,6 +75,14 @@ export const footer_nav = [
     path: '/playgrounds',
     Page: Playgrounds,
   },
+
+  {
+    id: 'Privacy Policy',
+    icon: <i className="devicon-vscode-plain" style={{fontSize: 20}}/>,
+    path: '/privacypolicy',
+    Page: PrivacyPolicy,
```

Comment:
The Privacy page should not have to be its own page. We can put a modal somewhere else like maybe the navbar.